### PR TITLE
Default config values + new java version

### PR DESCRIPTION
**Pull request by ohadbitt**
#### Pull Request Description

Using the config file we just added does not work on databricks if the library is taken from a local JAR
---

#### Future Release Comment

**Features:**
- new java version

**Fixes:**
- Default config values (used for parsing cluster URL)

### CODE REVIEW

File: connector/src/main/scala/com/microsoft/kusto/spark/utils/KustoDataSourceUtils.scala [hunk 1]
```diff
@@ -38,14 +38,17 @@ object KustoDataSourceUtils {
...
+    version = prop.getProperty("version")
+  }
   val clientName = s"Kusto.Spark.Connector:$version"
-  val ingestPrefix: String = prop.getProperty("ingestPrefix")
-  val enginePrefix: String = prop.getProperty("enginePrefix")
-  val defaultDomainPostfix: String = prop.getProperty("defaultDomainPostfix")
-  val defaultClusterSuffix: String = prop.getProperty("defaultClusterSuffix")
-  val ariaClustersPrefix: String = prop.getProperty("ariaClustersPrefix")
-  val ariaClustersSuffix: String = prop.getProperty("ariaClustersSuffix")
+  val ingestPrefix: String = prop.getProperty("ingestPrefix","http://ingest-")
```
Comment (vladikbr) [inline connector/src/main/scala/com/microsoft/kusto/spark/utils/KustoDataSourceUtils.scala:46]: here and everywhere else: https

File: connector/src/main/scala/com/microsoft/kusto/spark/utils/KustoDataSourceUtils.scala [hunk 2]
```diff
@@ -38,14 +38,17 @@ object KustoDataSourceUtils {
...
+  val enginePrefix: String = prop.getProperty("enginePrefix","http://")
+  val defaultDomainPostfix: String = prop.getProperty("defaultDomainPostfix","core.windows.net")
+  val defaultClusterSuffix: String = prop.getProperty("defaultClusterSuffix","kusto.windows.net")
+  val ariaClustersPrefix: String = prop.getProperty("ariaClustersPrefix", "https://kusto")
```
Comment (vladikbr) [inline connector/src/main/scala/com/microsoft/kusto/spark/utils/KustoDataSourceUtils.scala:50]: where is this used?
Can't we just use the Aria proxy full URI: https://kusto.aria.microsoft.com ?

Comment (ohadbitt) [reply connector/src/main/scala/com/microsoft/kusto/spark/utils/KustoDataSourceUtils.scala:50]: ok lets

File: connector/src/main/scala/com/microsoft/kusto/spark/utils/KustoDataSourceUtils.scala [hunk 3]
```diff
@@ -38,14 +40,17 @@ object KustoDataSourceUtils {
...
-  val defaultDomainPostfix: String = prop.getProperty("defaultDomainPostfix")
-  val defaultClusterSuffix: String = prop.getProperty("defaultClusterSuffix")
-  val ariaClustersPrefix: String = prop.getProperty("ariaClustersPrefix")
-  val ariaClustersSuffix: String = prop.getProperty("ariaClustersSuffix")
+  val ingestPrefix: String = prop.getProperty("ingestPrefix","ingest-")
+  val enginePrefix: String = prop.getProperty("enginePrefix","https://")
+  val defaultDomainPostfix: String = prop.getProperty("defaultDomainPostfix","core.windows.net")
+  val defaultClusterSuffix: String = prop.getProperty("defaultClusterSuffix","kusto.windows.net")
+  val ariaClustersProxy: String = prop.getProperty("ariaClustersProxy", "https://kusto.aria.microsoft.com")
+  val ariaClustersAlias: String = "aria"
```
Comment (vladikbr) [inline connector/src/main/scala/com/microsoft/kusto/spark/utils/KustoDataSourceUtils.scala:53]: Call this "Aria Proxy"

**Review by vladikbr — APPROVED** [review]

**Pull request merged**